### PR TITLE
Add github CI integration

### PR DESCRIPTION
**Pull request by dvzrv**
.github/workflows/build.yml:
Add simple github CI integration that tests configure, build and install
in an Arch Linux container.

### CODE REVIEW

**Comment by dvzrv** [comment]
@sadko4u not 100% sure, but you might have to configure this to be enabled in the project's configuration

**Comment by dvzrv** [comment]
You can watch the progress with CI on my branch (where the CI rule now triggers): https://github.com/dvzrv/lsp-plugins-art-delay/tree/github_ci

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+
+name: Build on various Operating Systems
+
+on:
+  push:
+    branches: [master, github_ci]
```
Comment (dvzrv) [inline .github/workflows/build.yml:7]: The `github_ci` branch still needs to be removed before merging!

Comment (sadko4u) [inline .github/workflows/build.yml:7]: The main development branch is `devel`. I think i is also reasonable to add it here.

Comment (dvzrv) [reply .github/workflows/build.yml:7]: Do you integrate (merge) towards devel?

Comment (sadko4u) [reply .github/workflows/build.yml:7]: The usual workflow is:
- `devel` branch collects all the changes that will be issued in the release.
- when release happens, all stuff becomes merged into `master` and the merge commit becomes tagged with the release version.
- The new `devel` becomes set up above the `master` branch with the incremented version number.

So I think it is more important to observe the changes on the `devel` branch rather on the `master` branch (but `master` also could be good, too).

Comment (dvzrv) [reply .github/workflows/build.yml]: I've added it both for branches and pull requests now.

**Comment by dvzrv** [comment]
I've also added carla and valgrind for @falkTX's follow up PR.

**Comment by dvzrv** [comment]
Okay, I believe this is now good to go!

**Comment by dvzrv** [comment]
@sadko4u I have changed base to devel now as well, so you'll be able to directly merge it.

**Comment by sadko4u** [comment]
I've merged the commit. How should I now observe that CI started to work?

**Comment by sadko4u** [comment]
Okay, I already see the output:
https://github.com/lsp-plugins/lsp-plugins-art-delay/runs/7024618337?check_suite_focus=true

**Comment by sadko4u** [comment]
@dvzrv Now I can propagate the changes for all other plugins. Do not do any PRs for that.

**Comment by dvzrv** [comment]
> @dvzrv Now I can propagate the changes for all other plugins. Do not do any PRs for that.

Cool. If you want to wait for @falkTX to add the valgrind/carla checks, then you'll only have to do it once.

**Comment by sadko4u** [comment]
How can I obtain available container images? I probably would like to add builds for `openSUSE Leap` and `Debian`, too.

**Comment by dvzrv** [comment]
> How can I obtain available container images? I probably would like to add builds for `openSUSE Leap` and `Debian`, too.

I guess that they use e.g. docker hub. You can look it up here: https://hub.docker.com/

My guess is, that e.g. Debian provides this https://hub.docker.com/_/debian and opensuse leap this https://hub.docker.com/r/opensuse/leap

**Pull request merged**